File: app/src/main/java/com/example/owner_pc/animechecker/model/entity/Characters.java
package com.example.owner_pc.animechecker.model.entity;

import java.io.Serializable;
import java.util.ArrayList;

/**
 * Created by owner-PC on 2017/05/20.
 */

public class Characters implements Serializable {
    public int id;
    public String nameFirst;
    public String nameLast;
    public String imageUrlLge;
    public String imageUrlMed;
    public String role;
    public int linkId;
    public ArrayList<StaffSmall> actor;

    @Override
    public String toString() {
        return "Characters{" +
                "id=" + id +
                ", nameFirst='" + nameFirst + '\'' +
                ", nameLast='" + nameLast + '\'' +
                ", imageUrlLge='" + imageUrlLge + '\'' +
                ", imageUrlMed='" + imageUrlMed + '\'' +
                ", role='" + role + '\'' +
                ", linkId=" + linkId +
                ", actor=" + actor +
                '}';
    }
}

File: app/src/main/java/com/example/owner_pc/animechecker/model/entity/Characters.kt
package com.example.owner_pc.animechecker.model.entity

import java.io.Serializable
import java.util.ArrayList

/**
 * Created by owner-PC on 2017/05/20.
 */

class Characters : Serializable {
    var id: Int = 0
    var nameFirst: String? = null
    var nameLast: String? = null
    var imageUrlLge: String? = null
    var imageUrlMed: String? = null
    var role: String? = null
    var linkId: Int = 0
    var actor: ArrayList<StaffSmall>? = null

    override fun toString(): String {
        return "Characters{" +
                "id=" + id +
                ", nameFirst='" + nameFirst + '\'' +
                ", nameLast='" + nameLast + '\'' +
                ", imageUrlLge='" + imageUrlLge + '\'' +
                ", imageUrlMed='" + imageUrlMed + '\'' +
                ", role='" + role + '\'' +
                ", linkId=" + linkId +
                ", actor=" + actor +
                '}'
    }
}
